src/rust/ensogl/src/display/symbol/dom.rs

```diff
@@ -28,7 +28,7 @@ mod js {
 
         export function set_object_transform(dom, matrix_array) {
             let css = arr_to_css_matrix3d(matrix_array);
-            dom.style.transform = 'translate(-50%, -50%)' + css;
+            dom.style.transform = css;
```

Comment:
Why this change? Wasn't it just placing DOM elements in such way that their origin is in the middle? If so, that was intentional - to make these shapes behave the same way as our webgl shapes.

Reply:
I think I had to do this for them to be consistent with the shapes. I will revisit this and double check the behaviour.